frontend/src/components/Patchnotes.jsx

```diff
@@ -77,22 +173,54 @@ const Patchnotes = ({ game }) => {
     );
 
     return (
-        <div>
-            {game === 'overwatch' ? (
-                <>
-                    {renderPatchNotes('Tank', patchDetails.Tanks)}
-                    {renderPatchNotes('Damage', patchDetails.Damages)}
-                    {renderPatchNotes('Support', patchDetails.Supports)}
-                    {renderPatchNotes('Map', patchDetails.Maps)}
-                    {renderPatchNotes('Bug', patchDetails.Bugs)}
-                </>
-            ) : (
-                <>
-                    {renderPatchNotes('Agent', patchDetails.Agents)}
-                    {renderPatchNotes('Map', patchDetails.Maps)}
-                    {renderPatchNotes('Bug', patchDetails.Bugs)}
-                </>
-            )}
+        <div className='patchnotes'>
+            <Navbar />
+            <div className='patches'>
+                {game === 'overwatch' ? (
+                    <>
+                        {renderPatchNotes('Tank', patchnotes.Tanks)}
+                        {renderPatchNotes('Damage', patchnotes.Damages)}
+                        {renderPatchNotes('Support', patchnotes.Supports)}
+                        {renderPatchNotes('Map', patchnotes.Maps)}
+                        {renderPatchNotes('Bug', patchnotes.Bugs)}
+                    </>
```

Comment:
Is this all a placeholder for now?

Reply:
No. This is how I planned on rendering game specific elements and giving them correct category headers. Please let me know if this could cause issues.